esp/scm/ws_dfu.ecm

```diff
@@ -812,6 +821,30 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
+ESPrequest [nil_remove] DFUCreateAndPublishRequest
+{
+    string LogicalName;
+    string Cluster;
+    string JobDescription;
+    string ECLRecordDefinition;
+    
+    bool ReturnJsonTypeInfo(false);
+    bool ReturnBinTypeInfo(false);
+    ESPenum SecAccessType AccessType;
```

Comment:
We might not need this unless @jakesmith 's dafileserv will require the user to provide "Write" access as part of the request.